src/components/ExerciseQuestion.tsx

```diff
@@ -19,7 +19,7 @@ export interface ExerciseQuestionProps {
   is_completed: boolean;
   correct_answer_id: ID;
   incorrectAnswerId: ID;
-  answerId?: ID;
+  answer_id?: ID;
```

Comment:
I renamed this so we can just get it from the spread instead of checking existence, feels simpler this way but don't feel that strongly about it if we'd rather not change this.